Application/SchedulingLib/SchedulingContext.cs

```diff
@@ -8,7 +8,7 @@ namespace SchedulingLib
 {
     class SchedulingContext : DbContext
     {
-        public DbSet<Instructor> Instructors { get; set; }
+        public DbSet<Instructors> Instructors { get; set; }
```

Comment:
You'll need to add the other classes in a similar way.
```suggestion
        public DbSet<Building> Buildings { get; set; }
        public DbSet<Room> Rooms { get; set; }
        public DbSet<Section> Sections { get; set; }
        public DbSet<Course> Courses { get; set; }
        public DbSet<Instructor> Instructors { get; set; }
```